lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -140,6 +145,70 @@ def test_unicode_user_partitions(self):
         _groups = [group.name for group in self.course.user_partitions[0].groups]
         self.assertEqual(_groups, user_groups)
 
+    def test_content_group_partition(self):
+        """
+        Test that cohort-schemed user partitions are ignored in the
+        grades export.
+        """
+        # Set up a course with 'cohort' and 'random' user partitions.
+        content_group_partition = UserPartition(
+            0,
+            'Content Group Configuration',
+            'Group Configuration for Content Groups',
+            [Group(0, 'Content Group A'), Group(1, 'Content Group B')],
+            scheme_id='cohort'
+        )
+        experiment_group_a = Group(2, 'Experiment Group A')
+        experiment_group_b = Group(3, 'Experiment Group B')
+        experiment_partition = UserPartition(
+            1,
+            'Content Experiment Configuration',
+            'Group Configuration for Content Experiments',
+            [experiment_group_a, experiment_group_b],
+            scheme_id='random'
+        )
+        course = CourseFactory.create(user_partitions=[content_group_partition, experiment_partition])
+
+        # Create user_a and user_b which are enrolled in the course
+        # and assigned to experiment_group_a and experiment_group_b,
+        # respectively.
+        user_a = UserFactory.create(username='user_a')
+        user_b = UserFactory.create(username='user_b')
+        CourseEnrollment.enroll(user_a, course.id)
+        CourseEnrollment.enroll(user_b, course.id)
+        course_tag_api.set_course_tag(
+            user_a,
+            course.id,
+            RandomUserPartitionScheme.key_for_partition(experiment_partition),
+            experiment_group_a.id
+        )
+        course_tag_api.set_course_tag(
+            user_b,
+            course.id,
+            RandomUserPartitionScheme.key_for_partition(experiment_partition),
+            experiment_group_b.id
+        )
+
+        # Verify that we see user_a and user_b in their respective
+        # content experiment groups, and that we do not see any
+        # content groups.
```

Comment:
Yes, I am not checking "we do not see any content groups," though now that I think about it, I don't think we should be checking that.

The bug which is verified as fixed by this test was that the mere existence of a cohort-schemed user partition caused the whole grading CSV upload to fail. Given that, I chose to only test that the existence of such a user partition no longer causes the task to fail. It would take additional setup to add cohorts and create a mapping between the cohorts and the user partition. I'm still happy to add it, though.